app/app/(tabs)/_layout.tsx

```diff
@@ -25,7 +26,7 @@ export default () => {
       }}
     >
       <Tabs.Screen
-        name="App"
+        name="HomePage"
```

Comment:
Great work, @salamf! I tested your work using the Xcode simulator on iOS devices (iPhone 15) and Android Studio on Android devices. However, there is an inconsistency in the navigation bar layout - there is a missing icon on the Android device, whereas it is visible on iOS. Additionally, I think we need to add a few images of food items and the names of restaurants to the homepage.
<img width="727" alt="Screenshot 2024-02-13 at 12 37 13 PM" src="https://github.com/SENG480a-NutriDine/mobile/assets/92160564/15ac7c45-5921-47f1-b6a9-f2bdace1b68f">

Reply:
I think that last icon is a placeholder indicating that there is no icon for that page (that icon leads to a test page that we will remove) - that's why its different on iOS and android.